geode-apis-compatible-with-redis/src/test/java/org/apache/geode/redis/internal/data/RedisSortedSetTest.java

```diff
@@ -139,46 +140,52 @@ public void equals_returnsTrue_givenDifferentEmptySortedSets() {
   }
 
   @Test
-  public void zadd_stores_delta_that_is_stable() throws IOException {
+  public void zadd_stores_delta_that_is_stable() {
     Region<RedisKey, RedisData> region = uncheckedCast(mock(Region.class));
+    when(region.put(any(), any())).thenAnswer(invocation -> {
+      RedisSortedSet value = invocation.getArgument(1, RedisSortedSet.class);
+      assertThat(value.hasDelta()).isTrue();
+      HeapDataOutputStream out = new HeapDataOutputStream(100);
+      value.toDelta(out);
+      ByteArrayDataInput in = new ByteArrayDataInput(out.toByteArray());
+      RedisSortedSet sortedSet2 = createRedisSortedSet("3.14159", "v1", "2.71828", "v2");
+      assertThat(sortedSet2).isNotEqualTo(value);
+      sortedSet2.fromDelta(in);
+      assertThat(sortedSet2).isEqualTo(value);
+      return null;
+    });
```

Comment:
This can be extracted to a method to avoid duplication.

Reply:
done